src/multihash.rs

```diff
@@ -85,47 +89,105 @@ pub struct RawMultihash {
     digest: crate::UnknownDigest<crate::U32>,
 }
 
-impl MultihashDigest for RawMultihash {
-    fn code(&self) -> u64 {
+impl RawMultihash {
```

Comment:
I'm not sure if RawMultihash not implementing the `MultihashDigest` trait anymore is a problem or not. I hope that the uses are so distinct from using a custom hash table that you won't run into weird type issues when using it.